tools/tpm2_pkcs11/utils.py

```diff
@@ -5,11 +5,16 @@
 import argparse
 import sys
 import shutil
+import yaml
 from tempfile import mkdtemp
 
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.ciphers import (Cipher, algorithms, modes)
 
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_PSS
```

Comment:
I guess this is pycrypto, which is likely not on the CI images, we should stick with the cryptography package.